app/src/main/java/org/dhis2/usescases/main/MainNavigator.kt

```diff
@@ -36,7 +36,7 @@ class MainNavigator(
     private var currentScreen: MainScreen? = null
     private var currentFragment: Fragment? = null
 
-    fun isHome(): Boolean = isPrograms() || isVisualizations()
+    fun isHome(): Boolean = isPrograms() && isVisualizations()
```

Comment:
It is not possible that `isProgram() && isVisualizations()` returns true as both checks that the variable currentScreen has a certainValue. Why did you changed the OR to AND?

Reply:
I see what you mean. However, when fixing the `show()` animation of the navigation bar, it would show it in the home screen without having the visualization feature activated. By doing this change the navigation is not shown in the home screen when the visualization feature is not activated.

Just saw that there is an PR that removes that feature and now it is activated. Maybe it's a good idea to check how this change affects the app now that the visualization is completely integrated in the app